akka-docs/src/main/paradox/typed/persistence-durable-state.md

```diff
@@ -0,0 +1,208 @@
+---
+project.description: Durable State with Akka Persistence enables actors to persist its state for recovery on failure or when migrated within a cluster.
+---
+# Durable State
```

Comment:
Just like `typed/persistence.md#tagging` there should be a section about tagging durable state changes.

Reply:
Yes, it's still in the works ..

Reply:
Now added ..